src/current/v23.2/node-shutdown.md

```diff
@@ -876,9 +876,9 @@ Cockroach Labs recommends that you:
 
 ## Decommissioning and draining on CockroachDB {{ site.data.products.advanced }}
 
-Most of the guidance in this page is most relevant to manual deployments, although decommissioning and draining work the same way behind the scenes in a CockroachDB {{ site.data.products.advanced }} cluster. CockroachDB {{ site.data.products.advanced }} clusters have a `server.shutdown.connections.timeout` of 1800 seconds (30 minutes) and a termination grace period that is slightly longer. The termination grace period is not configurable, and adjusting `server.shutdown.connections.timeout` is generally not recommended.
+Most of the guidance in this page is most relevant to manual deployments, although decommissioning and draining work the same way behind the scenes in a CockroachDB {{ site.data.products.advanced }} cluster. CockroachDB {{ site.data.products.advanced }} clusters have a termination grace period that is slightly longer than 30 minutes, and this grace period is not configurable.
 
-Client applications or application servers that connect to CockroachDB {{ site.data.products.advanced }} clusters should use connection pools that have a maximum lifetime that is shorter than the [`server.shutdown.connections.timeout`](#server-shutdown-connections-timeout) setting.
+Client applications or application servers that connect to CockroachDB {{ site.data.products.advanced }} clusters may adjust the the [`server.shutdown.connections.timeout`](#server-shutdown-connections-timeout) setting and should make sure the connection pool maximum lifetime is shorter than that value, as per the guidance in the rest of this page.
```

Comment:
```suggestion
You can adjust the [`server.shutdown.connections.timeout`](#server-shutdown-connections-timeout) setting for client applications or application servers that connect to CockroachDB {{ site.data.products.advanced }} clusters. Ensure that the connection pool maximum lifetime is shorter than that value, as per the guidance in [Termination grace period on Kubernetes](#termination-grace-period-on-kubernetes).
```

Reply:
Switched the link. I was actually looking past all that to the last line in the `#termination-grace-period-on-kubernetes` section, which says `A client application's connection pool should have a maximum lifetime that is shorter than the Kubernetes deployment's [server.shutdown.connections.timeout](https://www.cockroachlabs.com/docs/stable/node-shutdown#server-shutdown-connections-timeout) setting.`

But yeah, it's potentially confusing.

Reply:
lgtm!